RxJavaTemplate/app/src/debug/AndroidManifest.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="co.nimblehq.rxjava">
+
+    <application>
+        <activity
+            android:name=".EmptyHiltActivity"
+            android:exported="false" />
+    </application>
```

Comment:
Hmm. I am not sure if you tried to config the `sourceSets` in `gradle` rather than create debug folder for testing purpose 🤔  FYI: https://stackoverflow.com/a/56951415

Reply:
I'm unable to use `sourceSets` to move `AndroidManifest.xml` and `EmptyHiltActivity` to `androidTest` folder instead of `debug` 😅 I try every possible solution I found on the internet but none of which work as I always end up with unable to find Activity error, please let me know if I misunderstand something or if you have other solutions for this. 🙏🏼

Reply:
@chornerman I think you missed the step add `androidTest.java.srcDirrs` in `build.gradle(:app)` 🤔  I've just tried and it works. Please try on your side and let me know if it works 🙏 

- Remove debug folder and move the files to `androidTest` folder

![image](https://user-images.githubusercontent.com/6950766/122329601-ea037200-cf5b-11eb-98e0-f03a5b81a45e.png)

- Add `sourceSets` in `build.gradle(:app)`

![image](https://user-images.githubusercontent.com/6950766/122329703-0f907b80-cf5c-11eb-8f4b-4172c2012872.png)

Reply:
It's still not working 😢 
<img width="801" alt="Screenshot 2021-06-17 at 12 26 06" src="https://user-images.githubusercontent.com/13492460/122336979-dc53e980-cf67-11eb-95dc-c5d09046c6ae.png">

Reply:
@chornerman Ah. I executed test on `HomeFragmentTest`, didn't notice that `HomeFragmentTest` you are using `launchActivity` instead of `launchFragment`. As they commented from their sample, we have to put these files in `debug` folder 😞 

I'm curious why `HomeFragmentTest` but we launch activity instead of launch fragment as `SecondFragmentTest` and `WebViewFragmentTest`

Reply:
@hoangnguyen92dn Actually I follow [this](https://github.com/android/architecture-samples/blob/dev-hilt/app/src/androidTest/java/com/example/android/architecture/blueprints/todoapp/tasks/TasksFragmentTest.kt) approach. As I can recall when we call `launchFragment()`, it will launch the activity first and then open that fragment, but with `HomeFragment` it will launch the activity which will also open `HomeFragment` already since it's the main fragment and then open `HomeFragment` again as we call `launchFragment()` which will cause duplicate id problem when trying to match the view id 🙏🏼

Reply:
Refactored by removing tests that needed to navigate to other screens to `AppFlowTest`. Now `HomeFragmentTest` use `launchFragment` like the others 🙏🏼